src/btreemap/node/tests.rs

```diff
@@ -0,0 +1,39 @@
+use super::*;
+use crate::types::NULL;
+use proptest::collection::btree_map as pmap;
+use proptest::collection::vec as pvec;
+use std::cell::RefCell;
+use std::collections::BTreeMap;
+use std::rc::Rc;
+use test_strategy::proptest;
+
+fn make_memory() -> Rc<RefCell<Vec<u8>>> {
+    Rc::new(RefCell::new(Vec::new()))
+}
+
+#[proptest]
```

Comment:
In the original `proptest!` macro, code formatting doesn't work and has to be formatted manually, which I always find annoying. This attribute macro from the `test_strategy` crate is a wrapper around `proptest`, but it has the advantage that code formatting still works, along with other syntax simplifications.